Makefile

```diff
@@ -1,6 +1,6 @@
-KUBE_VERSION ?= 1.23
+KUBE_VERSION ?= 1.24
```

Comment:
We should probably make renovate responsible for bumping this version and tie it with https://github.com/kubernetes-sigs/kind/ releases as we use images from that project to tests our stuff.

Reply:
per our talk today, I am going to leave this be, being that we can't get renovate to pick this up yet.